docs/source/DeveloperDocumentation/PackagingGuides/bento4.md

```diff
@@ -1 +1,84 @@
-# Bento4
\ No newline at end of file
+# Bento4
+
+## Installing Bento4
+
+You can download the binaries for Bento4 for your preferred OS at the following link - [Bento4 binaries](https://www.bento4.com/downloads/)
```

Comment:
General website standards, from places like [w3](https://www.w3.org/QA/Tips/noClickHere#:~:text=Don't%20use%20%22click%20here%22%20as%20link%20text&text=When%20calling%20the%20user%20to,doesn't%20talk%20about%20mechanics), are against linking like this.
Something like the follow would be nicer:
```suggestion
You can download the binaries for your preferred OS from [Bento4](https://www.bento4.com/downloads/).
```